index.html

```diff
@@ -50,8 +55,14 @@ <h3>Let's CONNECT</h3>
         </section>
         <section class="portofolio">
             <div class="work-container">
```

Comment:
- [ ] For your entire Work Section kindly ensure that your images are alternating as we have on the template. For example, in the Figma template, the first image is on the left, the second on the right, the third on the left, the fourth on the right. Kindly implement that.

  Your Implementation | Figma Design
  :---:|:---:
  ![freddy-str](https://user-images.githubusercontent.com/36542986/161251224-d16bd24a-d460-41a3-b90f-690720c19a00.jpg) | ![freddy-alt](https://user-images.githubusercontent.com/36542986/161251218-97c87930-84db-4bbc-a0a9-f9467b9b33b5.jpg)

Reply:
Good job in implementing the changes.